projects/packages/jetpack-mu-wpcom/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.0 - 2023-02-15
```

Comment:
What's going on here?

Reply:
Since the package is now in use officially, I wanted to bump past the 0.x.x version. I think I tried doing that in a different PR by adding a major level changelog, but that didn't seem to work when a new version got created.

Reply:
Yeah, changelogger won't bump from 0.x to 1.0 automatically. But that should happen in a release PR, not some unrelated PR.

Reply:
Ok, I'll remember to bump the version when cutting the next release.